docs/README.md

```diff
@@ -22,78 +22,76 @@ Using your terminal, log in to the desired VTEX account and run the following co
 
-- `product-rating-inline`: Similar to the previous block (`product-rating-summary`), but intended to be used in [product shelves](https://vtex.io/docs/components/all/vtex.shelf/). The block displays the product's average rating only.
+- `product-rating-inline`: Similar to the previous block (`product-rating-summary`), but intended to be used in [product shelves](https://developers.vtex.com/docs/guides/vtex-shelf/). This block only displays the product average rating.
 
-## Modus Operandi
+## App behavior
 
-As described above, the app may be configured to use a **review moderation interface** where an administrator is responsible for approving the reviews before they are displayed on the store website.
+As described above, the app may be configured to use a **review moderation interface** where an admin is responsible for approving the reviews before they are displayed on the store.
 
 To access and use the review moderation admin interface, follow the instructions below:
 
 1. Enable the **Require Admin Approval** setting as described above.
-2. In the admin dashboard, navigate to **Catalog > Reviews** using the admin's sidebar.
-3. You may view either **Pending** or **Approved** reviews using the tabs at the top of the page.
+2. In the Admin dashboard, navigate to **Catalog > Reviews** using the Admin sidebar.
+3. You may view either **Pending** or **Approved** reviews using the tabs at the top of the page.
 > ⚠️ Warning
 >
-> You can export reviews to XLS files from the **Download** tab, and it is limited to exporting 800 reviews at a time. Please use the date pickers to select the time range of reviews you want to export.
+> You can export reviews to XLS files from the **Download** tab. The export is limited to 800 reviews at a time. Please use the date pickers to choose the time range of reviews you want to export.
 
 > ⚠️ Warning
 >
-> If you have updated to version 3.x after using a prior version of **Reviews and Ratings**, you may see the **Migrate Data** button instead of a list of reviews in the **Catalog > Reviews** page. Clicking this button will migrate all existing review data from the previous storage solution (VBASE) to the new solution (Masterdata V2). Once the migration is finished, the page will automatically refresh, and the list of reviews will become available for you.
+> If you have updated to version 3.x after using a prior version of **Reviews and Ratings**, you may see the **Migrate Data** button instead of a list of reviews in the **Catalog > Reviews** page. Clicking this button will migrate all existing review data from the previous storage solution (VBASE) to the new solution (Masterdata V2). Once the migration is finished, the page will automatically refresh, and the list of reviews will become available.
```

Comment:
```suggestion
> ⚠️ If you have updated to version 3.x after using a prior version of **Reviews and Ratings**, you may see the **Migrate Data** button instead of a list of reviews in the **Catalog > Reviews** page. Clicking this button will migrate all existing review data from the previous storage solution (VBASE) to the new solution (Masterdata V2). Once the migration is finished, the page will automatically refresh, and the list of reviews will become available.
```